tests/ui/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Genuine question happy if right now we are not clear - but do we need to persist the package-lock.json file ... is that derived from this file ?

Reply:
The purpose of package-lock. json is to ensure that the same dependencies are installed consistently across different environments, such as development and production environments. It also helps to prevent issues with installing different package versions, which can lead to conflicts and errors